models/Group.ts

```diff
@@ -20,10 +20,13 @@ const GroupSchema = new Schema({
         type: String,
     },
     description: {
+        type: String,
+    },
+    about: {
         text: {
             type: String,
         },
-        Media: [{
+        media: [{
             type: String,
         }]
     },
```

Comment:
One thing is the category, in the future integration from the backend and the frontend for the `posts` page, the `category` will be required, and I believe it could lies in the `Group` schema, but let's leave it for later